torch/csrc/distributed/rpc/request_callback_no_python.cpp

```diff
@@ -228,12 +222,9 @@ void RequestCallbackNoPython::processBaseScriptRemoteCall(
                               ->constValue()
                               .toRRef());
   } else {
-    ownerRRef = ctx.getOrCreateOwnerRRef(rrefId, returnType);
+    ownerRRef = ctx.getOrCreateOwnerRRef(rrefId, jitFuture->elementType());
```

Comment:
I assume `runJitFunction` and `runJitOperator` can retrieve type correctly, so that we can directly the the `elementType` from this Future?

Reply:
I did look into how they determine the type for the Future that they return, and they seem to be doing some kind of introspection and type inference. Hence to me they seem to be doing the right thing, but I'm not an expert on JIT so I may perhaps miss some nuances?